lib/client-assets.php

```diff
@@ -880,6 +881,10 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		'titlePlaceholder'   => apply_filters( 'enter_title_here', __( 'Add title', 'gutenberg' ), $post ),
 	);
 
+	if ( ! empty( $disable_custom_colors ) && is_bool( $disable_custom_colors[0] ) ) {
```

Comment:
`if ( ! empty( $disable_custom_colors[0] )` - is enough, `empty` operator performs all required checks.
By the way, why this is an array?

Reply:
This code was simplified we now use the format add_theme_support( 'disable-custom-colors' ); instead of add_theme_support( 'disable-custom-colors', true ); so no arrays are involved and we are certain we always have a true or false value.